snapsplit/src/features/trip/[tripId]/snap/_components/tabView/BaseTabView.tsx

```
@@ -0,0 +1,110 @@
+'use client';
+
+import { useState } from 'react';
+import SortFilterBar from '../SortFilterBar';
+import PhotoGrid from '../PhotoGrid';
+import { UploadedImage } from '../../type';
+import SortBottomSheet from '../SortBottomSheet';
+import FilterBottomSheet from '../FilterBottomSheet';
+import { FilterState } from '../../type';
+
+{/* 테스트 데이터 */}
+const testImages: UploadedImage[] = [
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)